README.md

```diff
@@ -98,7 +98,7 @@ Add a vim-plug section to your `~/.vimrc` (or `~/.config/nvim/init.vim` for Neov
     - Automatically executes `filetype plugin indent on` and `syntax enable`.
       You can revert the settings after the call. e.g. `filetype indent off`, `syntax off`, etc.
 
-#### Example
+#### Example .vimrc file
```

Comment:
It's noted above, and Neovim uses a different name by default.
Maybe use `Example configuration` ?

Reply:
`Example configuration` is ok.

Reply:
How about `Example configuration file`?